### PR TITLE
Update configure-network-connections-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by HarveyHuang443**
@denisebmsft can you help confirm if ctldl.windows.com is needed for Cloud-Delivered Protection? That URL is expressed as a FQDN but there is no resolution to that URL. Moreover, I can't find this anything anywhere within our documentation about that URL. I can see that ctldl.windowsupdate.com is needed for Defender A/V, so was this a spelling mistake on our part. Are we able to get a SME to confirm? Thank you!!

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/configure-network-connections-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Make sure that there are no firewall or network filtering rules denying access t
 
 |Service and description|URL|
 |---|---|
-|Microsoft Defender Antivirus cloud-delivered protection service is referred as Microsoft Active Protection Service (MAPS).<p> Microsoft Defender Antivirus uses the MAPS service to provide cloud-delivered protection.|`*.wdcp.microsoft.com` <p> `*.wdcpalt.microsoft.com` <p> `*.wd.microsoft.com` <p> `ctldl.windows.com` |
+|Microsoft Defender Antivirus cloud-delivered protection service is referred as Microsoft Active Protection Service (MAPS).<p> Microsoft Defender Antivirus uses the MAPS service to provide cloud-delivered protection.|`*.wdcp.microsoft.com` <p> `*.wdcpalt.microsoft.com` <p> `*.wd.microsoft.com` <p> `ctldl.windowsupdate.com` |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-network-connections-microsoft-defender-antivirus.md]:
```suggestion
|Microsoft Defender Antivirus cloud-delivered protection service is referred to as Microsoft Active Protection Service (MAPS).<p> Microsoft Defender Antivirus uses the MAPS service to provide cloud-delivered protection.|`*.wdcp.microsoft.com` <p> `*.wdcpalt.microsoft.com` <p> `*.wd.microsoft.com` <p> `ctldl.windowsupdate.com` |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @HarveyHuangMSFT, not related to your change, but could you please also commit the additional suggestion? Many thanks.

**Comment by denisebmsft** [comment]
Thank you @HarveyHuangMSFT

**Comment by HarveyHuang443** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**